documentation/docs/25-build-and-deploy/40-adapter-node.md

```diff
@@ -51,8 +51,15 @@ npm install dotenv
 ...and invoke it before running the built app:
 
 ```diff
--node build
-+node -r dotenv/config build
+- node build
++ node -r dotenv/config build
+```
+
+If you use Node.js v20.6+, you can use the [`--env-file`](https://nodejs.org/en/learn/command-line/how-to-read-environment-variables-from-nodejs) flag instead:
+
+```diff
+- node build
++ node --env-file=.env --env-file=.env.production build
```

Comment:
I suspect it's uncommon to have two env files. It makes the command look rather long to show it that way by default. I think if we wanted to show both it might be better to do it the way the node docs do where it shows it with a single one first and then shows a second example with multiple flags
```suggestion
+ node --env-file=.env build
```

Reply:
Does SvelteKit bundle all `.env.*` files supported by Vite?

I personally have a separate `.env.production` file for the Node.js production server.

Which is why I've added `--env-file=.env.production` in the documentation code.

```dotenv
# (adapter-node) -----------------------------------------------------------------------------------

# Reference https://kit.svelte.dev/docs/adapter-node#environment-variables

# The URL of the deployment should be provided for the CSRF protection (check origin) to work.
# If not, the server will respond with a [403] Cross-site POST form submissions are forbidden.

ORIGIN="" # https://your-domain.here

# `CF-Connecting-IP` provides the client IP address connecting to Cloudflare to the origin web server.
# Reference https://developers.cloudflare.com/fundamentals/reference/http-request-headers/

ADDRESS_HEADER="CF-Connecting-IP"
```

As suggested, `--env-file=.env` would be much simple, and match the dotenv example.

```shell
# Dotenv does support custom config path
node -r dotenv/config your_script.js dotenv_config_path=/custom/path/to/.env dotenv_config_debug=true
```

Reply:
The way I'd expect it to be used is:
- there is no `.env` file checked in except for something like `.env.sample`, `.env.local`/`.env.development`, `.env.test`, etc.
- the app is built on a CI. The CI provides the `.env` file or environment variables

Reply:
Never considered using CI with the Node.js adapter.

Since the Node.js docs is linked, removing duplicate flags seems reasonable.